libminifi/include/utils/TestUtils.h

```diff
@@ -0,0 +1,80 @@
+/**
+ *
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include <string>
+
+#include "../../test/TestBase.h"
+
+#ifdef WIN32
+#include <direct.h>
+#define GetCurrentDir _getcwd
+#else
+#include <unistd.h>
+#define GetCurrentDir getcwd
+#endif
+
+namespace org {
+namespace apache {
+namespace nifi {
+namespace minifi {
+namespace utils {
+
+std::string getCurrentWorkingDir(void) {
```

Comment:
This exists in FileUtils or pathutils

Reply:
I cannot find it there:

```bash
doxygen -g && perl -pi -e "s/GENERATE_(HTML|LATEX)(\s*)=.*$/GENERATE_\1\2= NO/g; s/GENERATE_MAN(\s*)=.*$/GENERATE_MAN\1= YES/g; s;INPUT(\s*)=.*$;INPUT\1=$(find . -name FileUtils.h);g" Doxyfile && doxygen && man ./man/man3/*
```
<img width="813" alt="Screenshot 2020-06-04 at 17 08 52" src="https://user-images.githubusercontent.com/64011968/83774523-123c6d80-a686-11ea-8ed2-1d1f54e2b8e3.png">

Reply:
Misguided you a bit, sorry!
It's called:
```
utils::Environment::getCurrentWorkingDirectory()
```

Reply:
Thanks for the suggestion, I removed the redundant function.